datadog_checks_base/datadog_checks/base/utils/aws.py

```diff
@@ -40,5 +41,8 @@ def rds_parse_tags_from_endpoint(endpoint):
     else:
         tags.append('dbinstanceidentifier:' + identifier)
         tags.append('hostname:' + endpoint)
+        # Add the `host` tag as the instance endpoint. This is a more useful tag for metrics
+        # because the agent's hostname is not the actual host being monitored.
+        tags.append('host:' + endpoint)
```

Comment:
I'm a bit confused about `host` vs. `hostname`, since it looks like they have the same `endpoint` value. Are they supposed to be the same?

Reply:
They should be the same. The `hostname` is meant to mirror the RDS integration tag which you can see here: https://github.com/DataDog/dogweb/blob/prod/integration/amazon_rds/crawler_per_account.py#L589-L600

`host` is meant to align on a tag which customers are more familiar with and is standard throughout Datadog. It will create duplicate values for different tags, but this is a better user experience to align on the `host` tag so customers can use context menus to link to logs, etc.